doc/nrf/known_issues.rst

```diff
@@ -392,13 +393,13 @@ LTE Sensor Gateway fails to run
 
     west build --board nrf9160dk_nrf9160_ns@1.1.0
 
-.. rst-class:: v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0
+.. rst-class:: v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0
```

Comment:
Issue fixed for v2.3.0 (merged here https://github.com/nrfconnect/sdk-nrf/pull/9873)